hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -807,6 +810,57 @@ public RaftPeerId getRaftPeerId() {
     return this.raftPeerId;
   }
 
+  @VisibleForTesting
+  public RaftPeerId getRaftLeaderId() {
+    return this.getRaftLeader() == null ? null : this.getRaftLeader().getId();
+  }
+
+  @VisibleForTesting
+  public String getRaftLeaderAddress() {
+    if (this.getRaftLeader() == null) {
+      return null;
+    }
+    InetAddress leaderInetAddress = null;
+    try {
+      Optional<String> hostname =
+          HddsUtils.getHostName(getRaftLeader().getAddress());
+      if (hostname.isPresent()) {
+        leaderInetAddress = InetAddress.getByName(hostname.get());
+      }
+    } catch (IOException ex) {
+      LOG.error("OM Ratis LeaderInetAddress {} is unresolvable",
+          getRaftLeader().getAddress());
+    }
+    return leaderInetAddress == null ? null :
+        leaderInetAddress.toString();
+  }
+
+  @VisibleForTesting
+  public RaftPeer getRaftLeader() {
+    RaftPeerId leaderId = null;
```

Comment:
Can we either reuse `OzoneManagerRatisServer.getLeader()` or replace `getLeader()` with `getRaftLeader()`?  Do we need both?